libdnf5-plugins/rhsm/rhsm.cpp

```diff
@@ -0,0 +1,169 @@
+/*
+Copyright Contributors to the dnf5 project.
+
+This file is part of dnf5: https://github.com/rpm-software-management/dnf5/
+
+Libdnf is free software: you can redistribute it and/or modify
+it under the terms of the GNU Lesser General Public License as published by
+the Free Software Foundation, either version 2.1 of the License, or
+(at your option) any later version.
```

Comment:
This license contradicts to COPYING.md where libdnf5-plugins code falls into GPLv2+. Please add "libdnf5-plugins/rhsm" to LGPLv2.1+ section in COPYING.md document.

Reply:
@ppisar 
> This license contradicts to COPYING.md where libdnf5-plugins code falls into GPLv2+. Please add "libdnf5-plugins/rhsm" to LGPLv2.1+ section in COPYING.md document.

That's not a bug in this PR. You have found a bug in COPYING.md. All libdnf5 library plugins are licensed with LGPLv2.1+. The code in "libdnf5-plugins" falls under GPLv2.1+.

Can you please do a PR with the COPYING.md fix?

Reply:
I see. Reported as issue #863.